src/fielddef.ts

```diff
@@ -576,6 +576,13 @@ const COMPATIBLE = {compatible: true};
 export function channelCompatibility(fieldDef: FieldDef<Field>, channel: Channel): {compatible: boolean; warning?: string;} {
   const type = fieldDef.type;
 
+  if (type === 'geojson' && channel !== 'shape') {
```

Comment:
Why does color not work?

Reply:
Why does it work?  Isn't geojson only for path shape?

Reply:
I thought geojson can have properties that can be encoded with the color of the shape for instance.

Reply:
For those properties, shouldn't the `type` be other typical type (not geojson)?

Reply:
Oh, I see. Yes.